### DISCUSSION TITLE
#🌟 加入我们的生电生草 Minecraft 服务器! 🌟

### DISCUSSION THREAD

**Discussion by dudu-ke** [share121/inter-knot#4933] [其他]
🌟 加入我们的生电生草 Minecraft 服务器! 🌟
在这里，您将体验到一个前所未有的 Minecraft 生存世界！我们的 生电生草服务器 等待着您和您的朋友们来探索、创造和冒险！

✨ 服务器特点：
生存模式：在这个真实的生存世界中，您需要采集资源，建立避难所，抵御夜晚的怪物，不断提升自己的生存能力。

自定义插件：我们精心挑选并安装了多种实用的插件，提升游戏体验，包括一键回家指令、玩家社区互动等，让您在Minecraft中能有更多的乐趣。

丰富的活动：参与我们的定期活动和比赛！无论是 翘翅飞行、建造比赛还是探险任务，总有适合您的活动，赢取丰厚的奖励和独特的称号！

温馨的社区：我们的玩家社区友好而积极！新手和老玩家们都可以在这里找到志同道合的伙伴，共同生存、探索和分享经验。

24/7 在线：我们的服务器全天候开放，无论您何时想来玩，随时都能加入我们的生存世界！

🛠️ 服务器信息：
服务器地址：gc.9666.fun:18042
版本：1.7.10—1.20.6可进（服务器为1.20.6）
加入方式：打开 Minecraft，选择“多人游戏”，点击“添加服务器”，输入服务器地址。
🎉 立即加入我们！
无论您是新手玩家，还是经验丰富的生存高手，我们服 服务器都将是您最理想的选择！赶快与朋友们一起来吧，一起建造属于您们的梦想世界！

🌈 期待在游戏中与您相见！<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

![2024-08-27_18 11 29](https://github.com/user-attachments/assets/4c8c0e63-65f6-479d-b09a-e13fc86807ce)

![2024-08-30_16 41 37](https://github.com/user-attachments/assets/ba232857-2477-4120-9cb7-7c1ee7c13f1f)

![2024-09-04_13 45 12](https://github.com/user-attachments/assets/a952cce0-3e4d-4b0e-97f7-94de88edaea4)

![2024-09-16_00 18 40](https://github.com/user-attachments/assets/615f3aba-3d75-4837-b642-12bb80e965f4)

![2024-09-16_00 20 48](https://github.com/user-attachments/assets/86489e56-4584-49c7-804b-3ea19f76138d)
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

![2024-09-07_20 37 06](https://github.com/user-attachments/assets/3eacbd6a-bd4d-4e98-a4da-079b08271cc2)

**Comment by weijiwj** [comment]
不错
